Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -166,6 +166,8 @@ public async Task<SubmissionDetailsServiceModel> GetDetailsById(int submissionId
             .OrderBy(tr => !tr.IsTrialTest)
             .ThenBy(tr => tr.OrderBy);
 
+        submissionDetailsServiceModel.MaxPoints = submissionDetailsServiceModel.Problem.MaximumPoints;
```

Comment:
Is it possible to do this using automapper?

Reply:
Fixed.